tests/output/common/as_widening.txt

```diff
@@ -1 +1 @@
-(Int | Str)("Hello")
+Hello
```

Comment:
The test no longer makes sense, but it can be deleted. Other tests should take care of what it used to do.